packages/design-system/src/components/OcInfoDrop/OcInfoDrop.vue

```diff
@@ -10,7 +10,11 @@
     <div class="info-drop-content">
       <div v-if="title" class="oc-flex oc-flex-between info-header oc-border-b oc-pb-s">
         <h4 class="oc-m-rm info-title" v-text="$gettext(title)" />
-        <oc-button appearance="raw">
+        <oc-button
+          v-oc-tooltip="$gettext('Close')"
+          appearance="raw"
+          :aria-label="$gettext('Close tooltip')"
```

Comment:
Not so sure about the `tooltip` wording. Isn't the generic `Close` sufficient?

Reply:
Had a quick chat with Tobi and that was what he suggested. I'm indifferent personally, not sure if the 'you are currentlly in a popover/tooltip' context gets communicated to e.g. a screenreader?